apps/ssr-tests/just.config.js

```diff
@@ -1,6 +1,6 @@
 const { preset, just } = require('@uifabric/build');
 const { task, series, logger } = just;
-const { spawn } = require('child_process');
+const { spawn } = require('just-scripts-utils');
```

Comment:
Curious the reasoning for the change. I assume this causes the build to fail if the spawn fails?

Reply:
Exactly. The `child_process` spawn returns a handle with a bunch of info, but `just` isn't looking at it in order to grok the result. This version gets wired into the just pipeline so that it does the right things.